python/dlisio/plumbing/basicobject.py

```diff
@@ -17,6 +17,19 @@ class BasicObject():
 
     attributes = {}
     linkage    = {}
+    """dict: Shows how to to interpret link objects stored in refs.
+    During load stage all the objects of obname, objref and attref types
+    are put into refs. During link stage they are linked to actually
+    loaded objects based on information provided in linkage.
+    If linkage is to be updated manually for the class:
+
+    >>> Coefficient.linkage['myparam'] = linkage.obname("PARAMETER")
```

Comment:
Maybe specify that the values **must** be either `linkage.obname("TYPE")` or `linkage.objref`?